viz/MLSMapVisualization.cpp

```diff
@@ -254,7 +316,31 @@ void MLSMapVisualization::updateMainNode ( osg::Node* node )
     geode->setShowNormals(showNormals);
     geode->setUncertaintyScale(uncertaintyScale);
 
-    p->visualize(*geode);
+    if (connectedSurface) {
```

Comment:
This should distinguish if the connected surface or the usual MLS patches are drawn right? Currently the connected surface is always drawn.

Reply:
When you use the viz test case to see the feature, the difference to the sloped map is not that big (look from below the surface or very close).
Perhaps also swiching to kalman shows the efect better